code/datums/json_savefile.dm

```diff
@@ -0,0 +1,71 @@
+/**
+ * A savefile implementation that handles all data using json
+ * also saves it using JSON too, fancy
```

Comment:
```suggestion
 * A savefile implementation that handles all data using json.
 * Also saves it using JSON too, fancy.
```

The doc website displays these as one line (I'm pretty sure), so should make it readable like a sentence